.github/workflows/toolshed.yaml

```diff
@@ -0,0 +1,25 @@
+name: Toolshed
+on: [push, pull_request]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
Should this use a postgres service like the previous Jenkins tests?

Reply:
Good point, that also revealed that the install model database (if created separately) and the tool shed databases were not created automatically. I've added this in https://github.com/galaxyproject/galaxy/pull/9528/commits/dd63ce043d48083bc40a2a96ebfa4ef74e8963af.